tsrc/cli/review.py

```diff
@@ -1,19 +1,83 @@
 """ Entrypoint for Review """
 
+from operator import itemgetter
+
 from .push import get_token, PushAction
 
+import ui
+
+def get_color_status(status):
+    color = ui.green
+    if status == 'failed':
+        color = ui.red
+    if status == 'pending' or status == 'created':
```

Comment:
prefer:

  if status in ['pending', 'created]:

Reply:
done